caravel/assets/visualizations/heatmap.js

```diff
@@ -10,8 +10,11 @@ require('./heatmap.css');
 // https://jsfiddle.net/cyril123/h0reyumq/
 function heatmapVis(slice) {
   function refresh() {
+    // Header for panel in explore v2
+    const header = document.getElementsByClassName('panel-title');
```

Comment:
sorry one more comment, are we sure there will only be one `panel-title` class on the page? will it be the first one?